ExpressionUtils/ParameterListRewriter.cs

```diff
@@ -11,9 +11,9 @@ namespace MiaPlaza.ExpressionUtils {
 	/// Rewrites a <see cref="LambdaExpression"/> to accept a single parameter <see cref="IReadOnlyList{T}"/> instead of multiple parameters, making handling of the compiled delegates much easier.

```

Comment:
I guess that is not true anymore. I guess we should find a new name for `ParameterListRewriter?